app/models/feedback.rb

```
@@ -1,4 +1,6 @@
 class Feedback < ApplicationRecord
+  enum service: {check: 0, ayts: 1}, _default: :check
```

I don't think we should specify a default - client code should be responsible for setting the appropriate value. We could pair this with a null false on the column so it fails loudly if not set.